test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala

```diff
@@ -42,6 +42,10 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
     }
   }
 
+  @JSExportTopLevel("org.scalajs.testinterface.HTMLRunner")
+  protected def getInstance(): this.type = this
+
+  @JSExport
```

Comment:
I was kind of dogfooding my own migration advice from the Scaladoc. But if you prefer I can change it. Anyway now I've seen that it works ;)